src/test/java/com/qa/restassured/TestRestAssured.java

```diff
@@ -0,0 +1,17 @@
+package com.qa.restassured;
+import static io.restassured.RestAssured.*;
+import static io.restassured.matcher.RestAssuredMatchers.*;
+import static org.hamcrest.Matchers.*;
+
+import org.testng.annotations.Test;
+
+public class TestRestAssured {
```

Comment:
Again name of class does not make any sense

Reply:
Modify